src/openrct2/world/map.h

```diff
@@ -260,25 +260,30 @@ enum
 };
 
 enum {
-    TILE_ELEMENT_SLOPE_NE_CORNER_UP = 1,
-    TILE_ELEMENT_SLOPE_SE_CORNER_UP = 2,
-    TILE_ELEMENT_SLOPE_SW_CORNER_UP = 4,
-    TILE_ELEMENT_SLOPE_NW_CORNER_UP = 8,
+    TILE_ELEMENT_SLOPE_FLAT = 0,
 
-    TILE_ELEMENT_SLOPE_NW_CORNER_DN = 7,
-    TILE_ELEMENT_SLOPE_SW_CORNER_DN = 9,
-    TILE_ELEMENT_SLOPE_SE_CORNER_DN = 13,
-    TILE_ELEMENT_SLOPE_NE_CORNER_DN = 14,
+    TILE_ELEMENT_SLOPE_ALL_CORNERS_UP = 15,
```

Comment:
@duncanspumpkin is this correct? Should it be a mask?

Reply:
It is used as both a mask and an actual item. I'm not sure what is clearest to read. IMHO I think its fine as you have it written.